stein buildtin jsonpath function cannot treat string beginning with number

## WHAT

As per title.

## How to reproduce

Change example yaml as below:

```diff
diff --git a/_examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml b/_examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml
index d4776bd..1245c4f 100644
--- a/_examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml
+++ b/_examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml
@@ -1,7 +1,7 @@
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
-  name: test
+  name: 2019-test
   namespace: x-echo-jp-dev
 spec:
   minAvailable: 50%
```
then apply,

```console
$ stein apply _examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml
_examples/manifests/microservices/x-echo-jp/development/PodDisruptionBudget/pdb.yaml
  [ERROR]  rule.filename                  Filename should be 2019.yaml (metadata.name + .yaml)

=====================
1 error(s), 0 warn(s)
[1]    99650 exit 1     stein apply
```

But error message is not correct. 

>Filename should be 2019.yaml (metadata.name + .yaml)

Error message should be this:

```
Filename should be 2019-test.yaml (metadata.name + .yaml)
```

--- a/lint/internal/policy/funcs/jsonpath.go
+++ b/lint/internal/policy/funcs/jsonpath.go
@@ -12,7 +12,6 @@ import (
 
 	"github.com/zclconf/go-cty/cty"
 	"github.com/zclconf/go-cty/cty/function"
-	ctyjson "github.com/zclconf/go-cty/cty/json"
 )
 
 // JSONPathFunc is
@@ -117,38 +116,15 @@ func GJSONFunc(file string, data []byte) function.Function {
 			Type: cty.DynamicPseudoType,
 		},
 		Type: func(args []cty.Value) (cty.Type, error) {
-			query := args[0].AsString()
-			defaultVal := cty.StringVal("")
-			if len(args) > 1 {
-				defaultVal = args[1]
-			}
-			res, err := getJSON(query, file, data)
-			if err != nil {
-				return defaultVal.Type(), nil
-			}
-			ty, err := ctyjson.ImpliedType(res)
-			if err != nil {
-				// When the result from getJSON can not be converted to JSON (that is, array or map),
-				// treat the return value as a string
-				return defaultVal.Type(), nil
-			}
-			return ty, nil
+			return cty.String, nil
 		},
 		Impl: func(args []cty.Value, retType cty.Type) (cty.Value, error) {
 			query := args[0].AsString()
-			defaultVal := cty.StringVal("")
-			if len(args) > 1 {
-				defaultVal = args[1]
+			result := gjson.GetBytes(data, query)
+			if !result.Exists() {
+				return cty.StringVal(""), nil
 			}
-			res, err := getJSON(query, file, data)
-			if err != nil {
-				return defaultVal, nil
-			}
-			val, err := ctyjson.Unmarshal(res, retType)
-			if err != nil {
-				return cty.StringVal(string(res)), nil
-			}
-			return val, nil
+			return cty.StringVal(result.String()), nil
 		},
 	})
 }

